### DISCUSSION TITLE
Custom Blocking URL

### DISCUSSION THREAD

**Discussion by kopacko** [TechnitiumSoftware/DnsServer#430] [Q&A]
Currently the only option is to use an IP address.

I want to just default the OpenDNS main blocked page URL :
https://block.opendns.com/

Optionally you can pass the domain in question :
https://block.opendns.com/main?url=test.com

What can I do to take advantage of this type of blocking setup?

**Comment by ShreyasZare** [comment]
Thanks for asking. If you mean to have a custom blocking page to show to the user when a domain name is blocked then you can do that using the Block Page DNS app. You can edit the web page in the app's www root to any custom html that you wish.
